configure

```diff
@@ -119,7 +119,7 @@ check_toolchain()
     if [ -z "$ARCH_INCLUDES" ]; then
         for dir in $(echo | $CC -Wp,-v -E - 2>&1 | grep '^ '); do
             local idir
-            idir="${dir}/$(uname -m)-linux-gnu/"
+            idir="${dir}/$($CC -print-multiarch)/"
```

Comment:
There should probably be a separate test for whether `$CC -print-multiarch` works at all. If `$CC` is set to clang, this command will fail, but then the dir check will succeed because the path expands to the parent dir and we'll end up duplicating all include dirs into `$ARCH_INCLUDES`...

Reply:
In my system(ARM based), it works for both gcc and clang.
```
linux:~$ gcc -print-multiarch
aarch64-linux-gnu
linux:~$ clang -print-multiarch
aarch64-linux-gnu
```